src/pages/index.js

```diff
@@ -80,12 +169,25 @@ addCardButton.addEventListener("click", () => {
 
 editProfileButton.addEventListener("click", function inputProfileInfo() {
   profFormPopup.open();
-  const currUserInfo = profUserInfo.getUserInfo();
-  profNameInput.value = currUserInfo.name;
-  profDescrInput.value = currUserInfo.description;
-  profileFormValidator.resetValidation();
+  const { name, description } = profUserInfo.getUserInfo();
+  profNameInput.value = name;
+  profDescrInput.value = description;
+  formValidators[profileModalForm.id].disableSubmitBtn();
```

Comment:
For the profile form opening you need `resetValidation` to remove errors from the inputs because the inputs have **valid** data you set in lines 173 -174


```js
  formValidators[profileModalForm.id].resetValidation();


```